tests/wdio/pageobjects/Setup.page.js

```diff
@@ -0,0 +1,37 @@
+const Page = require('../vaah-webdriverio/Page');
+const Sl = require('../vaah-webdriverio/Selector');
+const asserts = require('../vaah-webdriverio/Assert');
+
+class SetupPage extends Page{
+    constructor() {
+        super();
+        this.params.page.id = "SP";
+        this.params.page.name = "Setup";
+        this.params.page.path = "/backend#/setup";
+        this.params.page.url = this.base_url+this.params.page.path;
+
+    }
+
+    async open()
+    {
+        await browser.pause(this.is_human_pause)
```

Comment:
@we-abhijeet-g001 you should apply those pauses when `this.is_human` is true because if we're running those tests using ci/cd, those pauses will cause delay in execution

Reply:
this feedback is generalized